### ISSUE TITLE
Compiles empty data

### ISSUE THREAD

**Issue by AlexanderTserkovniy** [dehuszar/postcss-reference#2]
Hi,

Following this discussion https://github.com/postcss/postcss/issues/1091#issuecomment-349969709, I am trying to use your package and it seems like I am doing something wrong.

```css
@reference {
  @import "ui-components/src/common-styles/font.pcss";
}

:global {
  & .form-group {
    & label {
      @references .font-size-minion all;
    }
  }
}
```

Result:
http://prntscr.com/hkbtp0

What am I doing wrong?

Thanks.

**Comment by dehuszar** [comment]
Sorry for the slow response, somehow my Github email notifications aren't working.

Funny you should mention this.  I have been trying to debug it myself for a different use case.

You aren't doing anything wrong, per se.  Postcss-import has gone through some changes which don't play nice with what this plugin is doing.

From what I am able to glean, current versions of postcss-import won't pick up anything inside a custom AtRule as it's trying to conform to the CSS @import spec which requires all imports to be at the top of a CSS file.  You can see some discussion of this [here](https://github.com/postcss/postcss-import/issues/214#issuecomment-258257113).

I am trying to see if there is a way to allow postcss-import to still be usable inside the @reference AtRule, but in the meantime you can either use postcss-import v7.x (after which things break), or you can use postcss-nested-import which appears to work.

Let me know if that helps you.  I definitely want to resolve this.

**Comment by dehuszar** [comment]
@AlexanderTserkovniy 

Been a long time since you posted your issue, but I thought I'd let you know that this plugin has been rewritten and your issue should be resolved using v2.1.0.
